### PR TITLE
Document receiving HTTP Push OpenID RISC SETs (LG-3227)

### PR DESCRIPTION
**Pull request by zachmargolis**
This is my first draft at documenting support for this Security Events via HTTP Push, branch preview:
https://federalist-14ccf384-8bd5-4e30-aa95-a5fe5894e3c1.app.cloud.gov/preview/18f/identity-dev-docs/margolis-security-event-documentation/security-events/

Some open questions:
1. Where should this go in the nav? It's crowded as-is
2. Should this be combined with the [Push Notifications](https://developers.login.gov/push-notifications/) page?
   The documents are written in different styles, but there are a few key differences to the two APIs (sending/reciving), so it's worth discussing, and maybe making changes to unify both:  
   - the push framework uses the `iss-sub` (dash) key but this endpoint uses `iss_sub` per its spec
   - the push framework uses "WebPush" which puts all the content in an `Authorization` header, while this one uses "HTTP Push" which puts the content in the body and uses `Content-Type: application/secevent+jwt`

I also think I'm going to change the URL in the IDP to be under `/api/risc` instead of just `/api` so that change is incoming, as well as some other JTI fixes, now that I looked at my own implementation with fresher eyes

### CODE REVIEW

File: _pages/security-events.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+## Submitting a Security Event Token (SET)
+
+login.gov supports [Push-Based SET Token Delivery Using HTTP][push-http]. The SETs are signed [JWTs (JSON Web Tokens)](https://jwt.io/), similar to those used in the [OpenID Connect Authorization]({{site.baseurl}}/oidc/) flow.
+
+The [OpenID RISC Profile][openid-risc-events-profile] defines some very specific properties of these JWTs, and login.gov will validate against them, providing clear error messages where possible.
+
+### Request
+
+To submit an SET, send a HTTP POST request to the `/api/risc/security_events` endpoint. The body of the request is a signed JWT.
```
Comment (stevegsa) [inline _pages/security-events.md:52]: To submit a SET

Comment (stevegsa) [reply _pages/security-events.md:52]: send an

File: _pages/security-events.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+eyJpc3MiOiJ1cm46Z292OmdzYTpvcGVuaWRjb25uZWN0OnRlc3Q6cmlzYzpzZXRzIiwianRpIjoiYWJjZGVmZ2hpamtsbW5vcHFyc3R1dnd4eXoiLCJpYXQiOjE1OTU1MzIxNzgsImF1ZCI6Imh0dHBzOi8vaWRwLmludC5pZGVudGl0eXNhbmRib3guZ292L2FwaS9zZWN1cml0eV9ldmVudHMiLCJldmVudHMiOnsiaHR0cHM6Ly9zY2hlbWFzLm9wZW5pZC5uZXQvc2VjZXZlbnQvcmlzYy9ldmVudC10eXBlL2FjY291bnQtY3JlZGVudGlhbC1jaGFuZ2UtcmVxdWlyZWQiOnsic3ViamVjdCI6eyJzdWJqZWN0X3R5cGUiOiJpc3Nfc3ViIiwiaXNzIjoiaHR0cHM6Ly9pZHAuaW50LmlkZW50aXR5c2FuZGJveC5nb3YiLCJzdWIiOiIxMjNkNGY1Ni1qa2w3LTg5MTAxMS10MTJ2dy15MTNhMTQxNWQxNjE3Z2hpMTgifX19fQ
+.
+uN5dzVUnJvjFoFuh3F9yYwNDKie2kWkUOMpG7BEyVsObmsCPec1OFfshzBnnbdeUAyGMfhNyid11UsGraOumH5gFleXzwi6VrriGaHGblkUKwwlizUPSgmBVxqT237K7kmT_gHX-qXVGyLsqT_qUHP8of0JsKUkan2yAZHrnByFJjTLF5wy1K2sAMKN8TeMSxyFLRAppd_g3DYmE8jZWSk9tYfTP2sfaJE8P00BXCAAKzEhedneZbY8a7B9SruzWhSUy19zhc4Hd0f1aV_v9ckNnMa4IbQ573-qbSjCK0o1434XAJrZZj3psdjb2Spw44YkA1RQlPUngq1m0rOWVzA
+```
+
+### Response
+
+#### Successful response
+
+A successful SET submission will receive an empty response body with an 202 "Accepted" status.
```
Comment (stevegsa) [inline _pages/security-events.md:161]: with a

File: _pages/security-events.md (hunk 1 above)
Comment (zachmargolis) [reply _pages/security-events.md:52]: Great catches, thank you!

**Comment by zachmargolis** [comment]
Updated with new extended nav, thanks @juliaelman for the tip

screenie

**Pull request merged**